src/Core/src/Platform/Android/ViewGroupExtensions.cs

```diff
@@ -46,5 +47,11 @@ public static class ViewGroupExtensions
 
 			return null;
 		}
+
+		public static bool TryGetFirstChildOfType<T>(this AViewGroup viewGroup, [NotNullWhen(true)] out T? result) where T : AView
```

Comment:
The change looks good but you must modify the file https://github.com/dotnet/maui/blob/main/src/Core/src/PublicAPI/net-android/PublicAPI.Unshipped.txt to register the new option (public API). 

`/Users/builder/azdo/_work/1/s/src/Core/src/Platform/Android/ViewGroupExtensions.cs(51,22): error RS0016: Symbol 'TryGetFirstChildOfType<T>' is not part of the declared public API [/Users/builder/azdo/_work/1/s/src/Core/src/Core.csproj::TargetFramework=net7.0-android]`

Reply:
Had no idea, on it

Reply:
I just added it.